cmd/teams.go

```diff
@@ -101,7 +101,7 @@ func createTeamCmd(cliCtx *cli.Context) error {
 		return err
 	}
 
-	teamName, err := optionalArgLabel(cliCtx, 0, "team")
+	teamName, err := optionalArgName(cliCtx, 0, "team")
```

Comment:
Should be `optionalArgTitle` since this was wrong the first time.

Reply:
Thanks, will amend